### DISCUSSION TITLE
How to double-click a picture to pop up the picture view window

### DISCUSSION THREAD

**Discussion by qjzl** [ONLYOFFICE/DocumentServer#2816] [Q&A]
Pictures in Excel are generally placed in a reduced size. I hope that the picture box can be popped up by double-clicking or some other method, and the original picture can be viewed or the picture can be zoomed in or out.

This link is a functional demonstration of another web office product
https://github.com/user-attachments/assets/7cea9348-5cf0-4a87-b46f-e40a5dfaf166

**Comment by Gingetsu** [comment]
Hello @qjzl!
I created ticket as issue 69402 in our issues tracker.
Thank you for the enhancement!

**Reply by RoootRooot** [reply]
Sorry, I didn't see this "issues" listed.
